setup.py

```diff
@@ -21,11 +21,11 @@
 
 MIN_PYTHON_VERSION = "3.6"
 MIN_REQUIREMENTS = {
-    "numpy": "1.19.0",
+    "numpy": "1.19.3",
     "pandas": "1.1.0",
     "scikit-learn": "0.24.0",
     "statsmodels": "0.12.1",
-    "numba": "0.50",
+    "numba": "0.53",
```

Comment:
why is the change of `MIN_REQUIREMENTS` needed?

Reply:
There are no pre-built 3.9 wheels for 0.50 - 0.52: https://pypi.org/project/numba/0.52.0/#files. Pip uses `install_requires` to determine which dependency versions are installed when users pip install sktime, so this notes that numba 0.53 should be installed

Reply:
ok, got it - i.e., to simultaneously satsify 3.6-3.9 requirements.